addons/binding/org.openhab.binding.paradoxalarm/src/main/java/org/openhab/binding/paradoxalarm/internal/util/ParadoxUtil.java

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.paradoxalarm.internal.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ParadoxUtil} Utility class for different calculations / manipulations of data in the model and
+ * communicators.
+ *
+ * @author Konstantin_Polihronov - Initial contribution
+ */
+public class ParadoxUtil {
+
+    private static Logger logger = LoggerFactory.getLogger(ParadoxUtil.class);
```

Comment:
```suggestion
    private final Logger logger = LoggerFactory.getLogger(ParadoxUtil.class);
```

Reply:
I'll make it "static final" instead because it's called in a static way